.gitignore

```diff
@@ -0,0 +1,3 @@
+__pycache__
```

Comment:
I think these additions shouldn't be here. They should be in your global `.gitignore`.